### PR TITLE
fix: helm templating

### PR DESCRIPTION
**Pull request by zreigz**
- read from values.yaml
 - extent API versions for `group/version`
 - add unit test to check `.Capabilities.APIVersions.Has "monitoring.coreos.com/v1"` condition

### CODE REVIEW

File: pkg/manifests/template/helm.go [hunk 1]
```diff
@@ -168,7 +168,11 @@ func (h *helm) values(svc *console.GetServiceDeploymentForAgent_ServiceDeploymen
 		}
 	}
 
-	overrides, err := h.valuesFile(svc, "values.yaml.static")
+	filename := "values.yaml.static"
```
Comment (michaeljguarino) [inline pkg/manifests/template/helm.go:171]: what's the issue this is trying to protect against?

Comment (zreigz) [reply pkg/manifests/template/helm.go:171]: I have `values.yaml` file in my test scenario. I've noticed it was never taken into consideration. Finally my values map was always empty

Comment (michaeljguarino) [reply pkg/manifests/template/helm.go:171]: I feel like that's definitely unusual, i'm pretty sure base values.yaml is always there just not in the values overrides

Comment (zreigz) [reply pkg/manifests/template/helm.go]: I've changed my test `values.yaml` to `values.yaml.static`

**Review by seemywingz — APPROVED** [review]

**Pull request merged**